### PR TITLE
lelangovan's chennai file addition

### LINKED ISSUE #7: Create a new file for lelangovan Chennai, India
- [x] Create a branch 
- [x] Add commits
- [x] Open a PR
- [ ] Collaborate
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by lelangovan**
closes #7

### CODE REVIEW

**Comment by tanvipanchal** [comment]
👍

File: lelangovan-chennai.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Chennai, India
+
```
Comment (tanvipanchal) [inline lelangovan-chennai.md:2]: What's the weather like?

**Review by tanvipanchal — APPROVED** [review]
This is great! I have been to Chennai, its great ...

**Review by rdczowh — APPROVED** [review]
this is very good. approve..

**Pull request merged**